# My_research_publications

You can find the information about the published original research articles and a review article that I authored and co-authored with the abstract for each article.

The papers are arranged in chronological order, from latest to earliest.

Please feel free to explore the papers at your leisure. I'm happy to provide more details if you have any questions. Thank you!!

*First*

Pranav M Pande, Hamed Azarbad, Julien Tremblay, Marc St-Arnaud, Etienne Yergeau, 
Metatranscriptomic response of the wheat holobiont to decreasing soil water content, ISME Communications, Volume 3, Issue 1, December 2023, 30, 
https://doi.org/10.1038/s43705-023-00235-7

Abstract:
Crops associate with microorganisms that help their resistance to biotic stress. However, it is not clear how the different partners of this association react during exposure to stress. This knowledge is needed to target the right partners when trying to adapt crops to climate change. Here, we grew wheat in the field under rainout shelters that let through 100%, 75%, 50% and 25% of the precipitation. At the peak of the growing season, we sampled plant roots and rhizosphere, and extracted and sequenced their RNA. We compared the 100% and the 25% treatments using differential abundance analysis. In the roots, most of the differentially abundant (DA) transcripts belonged to the fungi, and most were more abundant in the 25% precipitation treatment. About 10% of the DA transcripts belonged to the plant and most were less abundant in the 25% precipitation treatment. In the rhizosphere, most of the DA transcripts belonged to the bacteria and were generally more abundant in the 25% precipitation treatment. Taken together, our results show that the transcriptomic response of the wheat holobiont to decreasing precipitation levels is stronger for the fungal and bacterial partners than for the plant.

*Second*

Gaike AHPaul D, Bhute SDhotre DP, Pande PUpadhyaya S, Reddy Y, Sampath R, Ghosh D, Chandraprabha D, Acharya J, Banerjee GSinkar VP, Ghaskadbi SS, Shouche YS. 2020. 
The Gut Microbial Diversity of Newly Diagnosed Diabetics but Not of Prediabetics Is Significantly Different from That of Healthy Nondiabetics. mSystems 5:10.1128/msystems.00578-19. 
https://doi.org/10.1128/msystems.00578-19

Abstract:
Type 2 diabetes (T2D) is a complex metabolic syndrome characterized by insulin dysfunction and abnormalities in glucose and lipid metabolism. The gut microbiome has been recently identified as an important factor for development of T2D. In this study, a total of 102 subjects were recruited, and we have looked at the gut microbiota of prediabetics (PreDMs) (n = 17), newly diagnosed diabetics (NewDMs) (n = 11), and diabetics on antidiabetic treatment (KnownDMs) (n = 39) and compared them with healthy nondiabetics (ND) (n = 35). Twenty-five different serum biomarkers were measured to assess the status of diabetes and their association with gut microbiota. Our analysis revealed nine different genera as differentially abundant in four study groups. Among them, Akkermansia, Blautia, and Ruminococcus were found to be significantly (P < 0.05) decreased, while Lactobacillus was increased in NewDMs compared to ND and recovered in KnownDMs. Akkermansia was inversely correlated with HbA1c and positively correlated with total antioxidants. Compared to ND, there was increased abundance of Megasphaera, Escherichia, and Acidaminococcus and decreased abundance of Sutterella in KnownDMs. Among many taxa known to act as community drivers during disease progression, we observed genus Sutterella as a common driver taxon among all diabetic groups. On the basis of the results of random forest analysis, we found that the genera Akkermansia and Sutterella and that the serum metabolites fasting glucose, HbA1c, methionine, and total antioxidants were highly discriminative factors among studied groups. Taken together, our data revealed that gut microbial diversity of NewDMs but not of PreDMs is significantly different from that of ND. Interestingly, after antidiabetic treatment, the microbial diversity of KnownDMs tends to recover toward that of ND.

*Third*

Bodkhe R, Shetty SA, Dhotre DP, Verma AK, Bhatia K, Mishra A, Kaur G, Pande P, Bangarusamy DK, Santosh BP, Perumal RC, Ahuja V, Shouche YS and Makharia GK (2019) 
Comparison of Small Gut and Whole Gut Microbiota of First-Degree Relatives With Adult Celiac Disease Patients and Controls. Front. Microbiol. 10:164. 
https://doi.org/10.3389/fmicb.2019.00164

Abstract:
Recent studies on celiac disease (CeD) have reported alterations in the gut microbiome. Whether this alteration in the microbial community is the cause or effect of the disease is not well understood, especially in adult onset of disease. The first-degree relatives (FDRs) of CeD patients may provide an opportunity to study gut microbiome in pre-disease state as FDRs are genetically susceptible to CeD. By using 16S rRNA gene sequencing, we observed that ecosystem level diversity measures were not significantly different between the disease condition (CeD), pre-disease (FDR) and control subjects. However, differences were observed at the level of amplicon sequence variant (ASV), suggesting alterations in specific ASVs between pre-disease and diseased condition. Duodenal biopsies showed higher differences in ASVs compared to fecal samples indicating larger disruption of the microbiota at the disease site. The duodenal microbiota of FDR was characterized by significant abundance of ASVs belonging to Parvimonas, Granulicatella, Gemella, Bifidobacterium, Anaerostipes, and Actinomyces genera. The duodenal microbiota of CeD was characterized by higher abundance of ASVs from genera Megasphaera and Helicobacter compared to the FDR microbiota. The CeD and FDR fecal microbiota had reduced abundance of ASVs classified as Akkermansia and Dorea when compared to control group microbiota. In addition, predicted functional metagenome showed reduced ability of gluten degradation by CeD fecal microbiota in comparison to FDRs and controls. The findings of the present study demonstrate differences in ASVs and predicts reduced ability of CeD fecal microbiota to degrade gluten compared to the FDR fecal microbiota. Further research is required to investigate the strain level and active functional profiles of FDR and CeD microbiota to better understand the role of gut microbiome in pathophysiology of CeD.

*Fourth*

Correa-García S, Pande P, Séguin A, St-Arnaud M, Yergeau E. 
Rhizoremediation of petroleum hydrocarbons: a model system for plant microbiome manipulation. Microb Biotechnol. 11(5):819-832. doi: 10.1111/1751-7915.13303. Epub 2018 Jul 31. PubMed PMID: 30066464; PubMed Central PMCID: PMC6116750.
https://doi.org/10.1111/1751-7915.13303

Abstract:
Phytoremediation is a green and sustainable alternative to physico-chemical methods for contaminated soil remediation. One of the flavours of phytoremediation is rhizoremediation, where plant roots stimulate soil microbes to degrade organic contaminants. This approach is particularly interesting as it takes advantage of naturally evolved interaction mechanisms between plant and microorganisms and often results in a complete mineralization of the contaminants (i.e. transformation to water and CO2). However, many biotic and abiotic factors influence the outcome of this interaction, resulting in variable efficiency of the remediation process. The difficulty to predict precisely the timeframe associated with rhizoremediation leads to low adoption rates of this green technology. Here, we review recent literature related to rhizoremediation, with a particular focus on soil organisms. We then expand on the potential of rhizoremediation to be a model plant-microbe interaction system for microbiome manipulation studies.

*Fifth*

Bhute S, Pande P, Shetty SA, Shelar R, Mane S, Kumbhare SV, Gawali A, Makhani H, Navandar M, Dhotre D, Lubree H, Agarwal D, Patil R, Ozarkar S, Ghaskadbi S, Yajnik C, Juvekar S, Makharia GK and Shouche YS (2016) 
Molecular Characterization and Meta-Analysis of Gut Microbial Communities Illustrate Enrichment of Prevotella and Megasphaera in Indian Subjects. Front. Microbiol. 7:660. 
https://doi.org/10.3389/fmicb.2016.00660

Abstract:
The gut microbiome has varied impact on the wellbeing of humans. It is influenced by different factors such as age, dietary habits, socio-economic status, geographic location, and genetic makeup of individuals. For devising microbiome-based therapies, it is crucial to identify population specific features of the gut microbiome. Indian population is one of the most ethnically, culturally, and geographically diverse, but the gut microbiome features remain largely unknown. The present study describes gut microbial communities of healthy Indian subjects and compares it with the microbiota from other populations. Based on large differences in alpha diversity indices, abundance of 11 bacterial phyla and individual specific OTUs, we report inter-individual variations in gut microbial communities of these subjects. While the gut microbiome of Indians is different from that of Americans, it shared high similarity to individuals from the Indian subcontinent i.e., Bangladeshi. Distinctive feature of Indian gut microbiota is the predominance of genus Prevotella and Megasphaera. Further, when compared with other non-human primates, it appears that Indians share more OTUs with omnivorous mammals. Our metagenomic imputation indicates higher potential for glycan biosynthesis and xenobiotic metabolism in these subjects. Our study indicates urgent need of identification of population specific microbiome biomarkers of Indian subpopulations to have more holistic view of the Indian gut microbiome and its health implications.
